aws/sdk/build.gradle.kts

```diff
@@ -284,7 +284,19 @@ fun generateCargoWorkspace(services: List<AwsService>): String {
     val generatedModules = services.map { it.module }.toSet()
     val examples = projectDir.resolve("examples")
         .listFiles { file -> !file.name.startsWith(".") }.orEmpty().toList()
-        .filter { generatedModules.contains(it.name) }
+        .filter { file ->
```

Comment:
Might be worth refactoring this into buildSrc, I think we even have a toml parser in tree already if you wanted to do it more directly

Reply:
Yeah, I agree. I'm going to hold off on it for this PR though, since I'm refactoring this area pretty heavily for splitting the release process.